### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dtreffenstaedt**
Updated README to reflect changes to build procedure since version 2.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +32,25 @@ Depending on the device of you choice, install the GUI for controlling the softw
...
+Follow the steps below to build the daemon:
 1. In folder `./muondetector-shared` excecute `qmake` and `make`. Depending on error messages check for missing dependencies.
 2. Copy the compiled libraries to your library folder: `sudo cp ./muondetector-shared/bin/* /usr/lib/`
 3. In folder `./muondetector-daemon` excecute `qmake` and `make`. Depending on error messages check for missing dependencies.
 4. The daemon can be found in folder `./muondetector-daemon/bin/`
 
+#### version >= 2.0.0
+Versions after this commit use CMake as build system. The steps to building the daemon are as follows:
+1. enter the build directory
+2. run `cmake ../source -DMUONDETECTOR_BUID_GUI=OFF`
```
Comment (marvin5300) [inline README.md]: is this '...BUID_GUI=OFF' correct? should be ON if you want to build the gui. Also there is a typo!?

File: README.md [hunk 2]
```diff
@@ -32,15 +32,25 @@ Depending on the device of you choice, install the GUI for controlling the softw
...
+3. run `make package`
+4. go to `../packages`
```
Comment (marvin5300) [inline README.md]: Confusion because now you are in build directory and packages should also be in build directory so you don't need ../ !?

**Review by marvin5300 — CHANGES_REQUESTED** [review]
Found two little issues.

File: README.md (hunk 2 above)
Comment (marvin5300) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (dtreffenstaedt) [reply README.md]: Until https://github.com/MuonPi/muondetector/pull/43 is merged in main the process is still slightly different I realised. The package does not get build properly when building from the main project file.

Comment (dtreffenstaedt) [reply README.md]: The current process should be laid out properly with https://github.com/MuonPi/muondetector/pull/42/commits/ebc9135c7766559a801d62db0f9ae669aa772133

**Review by hangeza — APPROVED** [review]

**Review by marvin5300 — APPROVED** [review]

**Pull request merged**